go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/h2non/gock v1.2.0
 	github.com/onsi/ginkgo/v2 v2.13.0
 	github.com/onsi/gomega v1.29.0
+	github.com/prometheus/client_golang v1.17.0
```

Comment:
I guess `go.sum` should be updated as well.

Reply:
it wasn't added but just moved from "indirect" up to direct, so go.sum has it